h3m/h3mlib/meta/meta_push_od.c

```diff
@@ -272,8 +272,16 @@ static int _meta_push_od_seers_hut(uint32_t fm,
         (struct H3M_OD_BODY_DYNAMIC_QUEST_GUARD *)body, meta_od_entry);
 
     n = sizeof_reward(fm, body->reward_type);
-    META_OBJECT_PUSH_PTR(meta_od_entry->dyn_pointers, body, body->reward, n, 0)
-
+	// Convert AB/SoD 2-byte amounts to RoE 1-byte 
```

Comment:
Wrong indentation but looks like a good addition, should be "else if" instead of if at line 280 though.